dozer-ingestion/src/connectors/postgres/iterator.rs

```diff
@@ -161,7 +164,12 @@ impl<'a> PostgresIteratorHandler<'a> {
                 ingestor: self.ingestor,
                 connector_id: self.connector_id,
             };
-            tables = snapshotter.sync_tables(details.tables.clone(), self.lsn.borrow().as_ref())?;
+            tables = snapshotter.sync_tables(details.tables.clone())?;
+
+            let lsn = self.lsn.borrow().map_or(0, |(lsn, _)| u64::from(lsn));
+            self.ingestor
+                .handle_message(((lsn, 0), SnapshottingDone))
```

Comment:
Is lsn guarenteed to be positive?

Reply:
Yes, it is coming from a slot creation query. LSN struct is `pub struct PgLsn(u64);`.

Reply:
And it's never `0`?

Reply:
Yes. LSN can be between 0/0 and FFFFFFFF/FFFFFFFF. So theoretically it can be 0, but before creating slot, we have already created publication, so we can assume that slot creation message cannot have LSN 0/0.

Reply:
Wonderful. Maybe put a comment line there so later we don't get confused again and believe this `0` is a potential bug?